scripts/ensmallen-release.sh

```diff
@@ -132,7 +133,7 @@ git add HISTORY.md;
 git commit -m "Update and release version $MAJOR.$MINOR.$PATCH.";
 
 changelog_str=`cat HISTORY.md |\
-    awk '/^### /{f=0} /^### ensmallen 2.13.0: "Automatically Automated Automation"/{f=1} f{print}' |\
+    awk '/^### /{f=0} /^### ensmallen $MAJOR.$MINOR.$PATCH: "$version_name"/{f=1} f{print}' |\
```

Comment:
```suggestion
    awk '/^### /{f=0} /^### ensmallen '"$MAJOR"'.'"$MINOR"'.'"$PATCH"': '"$version_name"'/{f=1} f{print}' |\
```
I believe that we'll have to do this in order to get the `MAJOR`, `MINOR`, and `PATCH` variables to substitute into the awk program correctly.  Correct me if I'm wrong on that.

Reply:
Ah my bad, I didn't test the script! 😛

Reply:
No worries!  If you want, you could test by running it, then just closing the PR that it opens. :+1:

Reply:
Ok sure! :)

Reply:
One thing of note though, when I created the PR, I had to be on the master branch of the upstream repo (cloned from mlpack, not my fork).

Reply:
Oh, right.  Well, in that case, we can try to merge this and I suppose we can test it then. :)

Reply:
Haha, okay! :)

Reply:
Yeah, we can always just close the test PR, and if it's wrong, we can make another attempt at fixing the script.  I realize I wrote it in such a way that it needs to be on the master branch, and that makes it a bit hard to debug. :)